### DISCUSSION TITLE
Consult with Autism Researchers

### DISCUSSION THREAD

**Discussion by GeorgiaHCA** [alan-turing-institute/AutSPACEs#365] [General]
We need to consult with a wide range of autism researchers to understand how to best build the platform from their perspective.

**Comment by ThomasAJR** [comment]
It may also be useful to consult with some administrators/teachers at an autism specific school? (I have a contact at the LSV Hassocks school, if you'd like to be connected with her.)  Teachers may have a unique view into how younger autistic persons prefer to engage with technology.

**Comment by EllenDevereux** [comment]
@GeorgiaHCA upload results of conversations that have been had.
These results can be used to build a persona for a researcher. @fjThomasStanley (could you please work with Chris Legg to achieve this).
